src/reporters/interfaces.d.ts

```diff
@@ -0,0 +1,140 @@
+declare module 'intern/lib/reporters/Runner' {
+	import * as charm from 'charm';
+	import Collector = require('istanbul/lib/collector');
+	import * as Reporter from 'istanbul/lib/report/text-summary';
+	import * as DetailedReporter from 'istanbul/lib/report/text';
+	import * as Suite from 'intern/lib/Suite';
+	import * as Test from 'intern/lib/Test';
+
+	class Runner {
+		protected sessions: { [sessionId: string]: any };
+		protected proxyOnly: boolean;
+		protected reporter: Reporter;
+		protected detailedReporter: DetailedReporter;
+		charm: charm.CharmInstance;
```

Comment:
It is being used in intern's `Runner` and there are situations where that underlying charm still gets invoked by the methods that aren't overridden in the custom `Reporter`.